problems/2.functions.js

```diff
@@ -20,7 +23,8 @@ function greeting() {
 * @returns {null}
 * @modifies {global.globalCount}
 */
-function incrementGlobalCount() {
+function incrementGlobalCount(val) {
+  
```

Comment:
global.globalCount += val

To update a variable you need to do reassignment and thus need the assignment operator (=).